package com.revature.p1.web.exceptions;

public class TradeAlreadyExistsException extends Exception {
	
	/*
	 * try {
	 * 
	 * } catch (Exception e) {
	 * System.out.println("Sorry but Trade can not be completed"); }
	 */
}
